core/src/test/scala/org/typelevel/paiges/PaigesScalacheckTest.scala

```diff
@@ -258,6 +258,16 @@ class PaigesScalacheckTest extends FunSuite {
     forAll { (a: Doc) => assert(okay(a)) }
   }
 
+  test("Union(a1, b1) + Union(a2, b2) === Union(a1 + a2, b1 + b2)") {
+    import Doc._
+    forAll { (d1: Doc, d2: Doc) =>
+      (d1.grouped, d2.grouped) match {
+        case (d1@Union(a1, b1), d2@Union(a2, b2)) => Concat(d1, d2) === Union(Concat(a1, a2), Concat(b1, b2))
```

Comment:
you have also left out the `assert` here, so we aren't actually testing anything. This is confusing, but when using scalacheck from scalatest you generally must use asserts.

Reply:
Oof, thanks!  I'm sad I didn't get a compiler warning here.  Would wartremover be something we can consider?

Reply:
Ug, ScalaTest just loves NonUnitStatements, so WartRemover doesn't help at all.